2200107/문제1.py

```diff
@@ -0,0 +1,18 @@
+# SWEA 11856번 반반
+# SWEA는 import sys 못쓰네; 오류남..
+import sys
+# TC=int(sys.stdin.readline())
+TC=int(input())
+
+for tc in range(1,TC+1):
+    # s=sys.stdin.readline().rstrip()
+    s=input()
+    ss=set(s)
+    # print(ss)
+    li_s=list(ss)
+    if len(ss)!=2:
+        print(f'#{tc} No')
+    elif len(li_s)==2:
+        if s.count(li_s[0])==2:
+            if s.count(li_s[1])==2:
+                print(f'#{tc} Yes')
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    if len(ss)!=2:
        print(f'#{tc} No')
    elif len(li_s)==2:
        if s.count(li_s[0])==2 and s.count(li_s[1])==2:
            print(f'#{tc} Yes')
        else:
            print(f'#{tc} No')
```
현재 코드는 `AAAB` 같이 한 문자가 3번 등장하는 경우에 `No`를 출력하지 못합니다! 위와 같은 방법으로 수정해주시면 해결할 수 있습니다 😄